doc/deployment/amazon_web_services/index.rst

```diff
@@ -0,0 +1,48 @@
+.. _deploy-aws:
+
+Deploy Pachyderm on Amazon AWS
+==============================
+
+Pachyderm can run in a Kubernetes cluster deployed in Amazon®
+Web Services (AWS), whether it is an Elastic Container
+Service (EKS) or a Kubernetes cluster deployed by a deployment tool.
+AWS removes the need to maintain the underlying virtual cloud.
```

Comment:
People know what AWS is, no? 

"AWS removes the need to maintain the underlying virtual cloud." Is of course a true statement, but I think is obvious. While I understand I'm not one to talk in this regard, let's err on the side of only writing what's needed and keeping the docs tighter.